woocommerce/google-listings-and-ads
# Attribute Mapping - Add Inbox Notification

### Changes proposed in this Pull Request:



Closes part of #1648 .

This PR add the inbox notification informing existing users about the new feature.

### Screenshots:

<img width="720" alt="Screenshot 2022-11-17 at 12 36 08" src="https://user-images.githubusercontent.com/5908855/202360853-1c54eaa9-8527-449d-b4a6-41b3437faf36.png">

### Detailed test instructions:


1. Be sure `gla_install_timestamp` in your `wp_options` table is `> 1669766400`
2. Go to WooCommerce->Status->AcctionSchedule->Pending and run the task `wc_gla_cron_daily_notes`
3. Go to WooCommerce->Home. The notification should NOT be added
4. Change `gla_install_timestamp` in your `wp_options` table to `<= 1669766400`
5. Go to WooCommerce->Status->AcctionSchedule->Pending and run the task `wc_gla_cron_daily_notes`
6. Go to WooCommerce->Home. The notification should YES be added
7. Verify the content match [this content](https://docs.google.com/document/d/1hVV-mFeSD4qsnIv-G6vFJ6zdYDMRZLiagquGqY_gpqg/edit). 
8. Click on Learn more. It should go to Attribute Mapping feature
9.  Go to WooCommerce->Status->AcctionSchedule->Pending and run the task `wc_gla_cron_daily_notes`
10. Go to WooCommerce->Home. The notification should NOT be added
